dist/images/install.sh

```diff
@@ -2935,6 +2937,52 @@ dbtool(){
         dbstatus)
           kubectl exec "$OVN_NB_POD" -n $KUBE_OVN_NS -c ovn-central -- ovn-appctl -t /var/run/ovn/ovnnb_db.ctl ovsdb-server/get-db-storage-status OVN_Northbound
           ;;
+        restore)
+          # set ovn-central replicas to 0
+          replicas=$(kubectl get deployment -n $KUBE_OVN_NS ovn-central -o jsonpath={.spec.replicas})
+          kubectl scale deployment -n $KUBE_OVN_NS ovn-central --replicas=0
```

Comment:
Should we wait for the operation to be completed?

Reply:
We need to move db file to /tmp in ovs-ovn pod. The ovs-ovn pod will be crashed if we wait enough time when I did test. That will result in the failure of the command.So don't wait here.